modules/custom/az_core/az_core.module

```diff
@@ -73,3 +73,12 @@ function az_core_block_access(Block $block, $operation, AccountInterface $accoun
   }
   return AccessResult::neutral();
 }
+/**
+ * Implements hook_theme_registry_alter().
+ */
+function az_core_theme_registry_alter(&$theme_registry) {
+  $key = array_search('paragraphs_preprocess_field_multiple_value_form', $theme_registry['field_multiple_value_form']['preprocess functions']);
```

Comment:
I would suggest the following changes:

- `array_search` should use the optional argument to indicate strict comparison.
- the code should check if the array key is set before array searching.